services/ATOMIC-SWAPS/COINS/LIST.json

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "name": "City Chain",
```

Comment:
Remove all extra metadata that is already accessible from other json files on the same repo.